### ISSUE TITLE
Can't install Logitech G Hub

### ISSUE THREAD

**Issue by Dim0N22** [sandboxie-plus/Sandboxie#955]
**Describe the bug**
Can't install Logitech G Hub.

**To Reproduce**
1. Go to https://www.logitechg.com/en-us/innovation/g-hub.html
2. Click on `DOWNLOAD FOR WINDOWS`
3. Run the installer in a sandbox with internet access enabled
4. See the installer is stuck

**Expected behavior**
Installer successfully installs the program.

**Screenshots**
![image](https://user-images.githubusercontent.com/1761368/124343781-18e82d80-db83-11eb-97ab-196252bf5327.png)

**System details and installed software (please provide the following information):**
 - What is your Windows edition and version? (i.e. Windows 10 Pro 20H2).
**Win10 Pro 21H1**

 - What is your current Sandboxie edition and version? (i.e. Sandboxie Plus 0.7.2).
**Sandboxie Plus 0.8.2**

 - Please mention in which version this bug got introduced, because it acts as a guide for testers and developers.
**Didn't try other versions.**

 - Please mention any security software running in the background, even if the real-time protection is disabled.
**Windows Defender.**

**Sandboxie configuration**
<details>
 
<summary>My sandboxie.ini settings</summary>
 
```
[GlobalSettings]

Template=RpcPortBindings
Template=7zipShellEx
Template=WindowsRasMan
Template=RTSS
Template=WindowsLive
Template=OfficeLicensing

[UserSettings_0BCC0198]

SbieCtrl_AutoStartAgent=SandMan.exe
SbieCtrl_UserName=Dmitry
SbieCtrl_NextUpdateCheck=1625552669
SbieCtrl_WindowCoords=212,626,2217,843
SbieCtrl_ActiveView=40021
SbieCtrl_ProcessViewColumnWidths=250,70,300
BoxDisplayOrder=Hardened,Apps(Logitech_GHub)
SbieCtrl_BoxExpandedView=,

[Logitech_GHub]

Enabled=y
AutoRecover=n
BlockNetworkFiles=y
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
RecoverFolder=%Personal%
RecoverFolder=%Desktop%
BorderColor=#00FFFF,ttl
Template=OpenBluetooth
Template=SkipHook
Template=FileCopy
Template=qWave
Template=BlockPorts
Template=LingerPrograms
Template=Chrome_Phishing_DirectAccess
Template=Firefox_Phishing_DirectAccess
Template=AutoRecoverIgnore
ConfigLevel=9
```

</details>

**Comment by Dyras** [comment]
Is there a reason in particular you want to install the.. uh... driver for your mouse in Sandboxie? This seems like something that should always be run on "bare metal"

**Comment by Dim0N22** [comment]
I definitely expect troubles with making it work, but in my understanding at least the installer should succeed or throw an error why it fails. 

The reason for installing it in Sandboxie is that I don't really need it all the time and I don't like that it creates roughly a GB of different files all over my disk. Ideally I want to have it all in a sandbox that I can just delete later.

Anyway, feel free to close it this behavior is expected. Thanks!

**Comment by isaak654** [comment]
The installation is completed at 100% on Classic 5.5.0.2 and Win 10 Pro 20H2, but the program's window of the main executable _lghub.exe_ is not loaded at all. Imho it should be still loaded graphically, even if the drivers are correctly filtered from Sandboxie.

**Comment by birdie-github** [comment]
I'm not sure applications using kernel drivers should be paid any attention to. This one in particular is completely useless without kernel drivers.

**Comment by isaak654** [comment]
> 
> 
> I'm not sure applications using kernel drivers should be paid any attention to. This one in particular is completely useless without kernel drivers.

Yeah, unless you want to see how it looks a program without installing it on the real system. That's still a use case to me.

**Comment by DavidXanatos** [comment]
> 
> Is there a different UAC elevation management in Classic and Plus? I only used clean sandboxes in both editions.

no its the same, strange :/

**Comment by isaak654** [comment]
@DavidXanatos I've updated the previous post about the new 0.8.6 release.
